### DISCUSSION TITLE
(1.20.1) Create Primed TNT that looks like a different block by changing its BlockState?

### DISCUSSION THREAD

**Discussion by StupidRepo** [FabricMC/fabric-api#4076] [Mod Dev Support] [answered]
### Description

I know in 1.20.6 you can call `.setBlockState(BlockState)` on a `TntEntity` however I can't seem to do this in 1.20.1.
Is there a way to do this in 1.20.1?

**Comment by haykam821** [comment, marked as answer]
Block state customization for TNT was added in [Minecraft snapshot 23w42a](https://www.minecraft.net/en-us/article/minecraft-snapshot-23w42a) for Minecraft 1.20.3, so you cannot use the same method in Minecraft 1.20.1. You can use a workaround like creating your own TNT entity with a custom entity renderer, or if you are writing a server-side mod, you can potentially use a block display with a text display that replicates the fuse flashing.
